forge-gui/res/cardsfolder/upcoming/anointed_peacekeeper.txt

```diff
@@ -0,0 +1,14 @@
+Name:Anointed Peacekeeper
+ManaCost:2 W
+Types:Creature Human Cleric
+PT:3/3
+K:Vigilance
+K:ETBReplacement:Other:ChoosePlayer
+SVar:ChoosePlayer:DB$ ChoosePlayer | Defined$ You | Choices$ Player.Opponent | ChoiceTitle$ Choose an opponent to look at the hand: | SubAbility$ DBLook | SpellDescription$ As CARDNAME enters the battlefield, look at an opponent's hand, then choose any card name.
+SVar:DBLook:DB$ RevealHand | Defined$ ChosenPlayer | Look$ True | SubAbility$ DBNameCard
+SVar:DBNameCard:DB$ NameCard | Defined$ You | SubAbility$ DBClear
+SVar:DBClear:DB$ Cleanup | ClearChosenPlayer$ True
+S:Mode$ RaiseCost | EffectZone$ Battlefield | ValidCard$ Card.NamedCard | Type$ Spell | Amount$ 2 | Activator$ Opponent | Description$ Spells your opponents cast with the chosen name cost {2} more to cast.
+S:Mode$ RaiseCost | EffectZone$ Battlefield | ValidCard$ Card.NamedCard | Type$ NonManaAbility | Amount$ 2 | Activator$ Opponent | AffectedZone$ Battlefield | Description$ Activated abilities of sources with the chosen name cost {2} more to activate unless they're mana abilities.
```

Comment:
Are we sure this only works on the battlefield?

it talks about Sources, not about "permanents"